### PR TITLE
explain why SenML and SensML

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Review by cabo — CHANGES_REQUESTED** [review]
Please see #118

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -1299,6 +1299,21 @@ media types for the same formats of the streaming use (SensML). Clipboard
 formats are defined for the JSON and XML form of lists but do not make
 sense for streams or other formats.
 
+The reason there are both SenML and the streaming SensML formats is
+that they are not the same data formats and they require separate
+negotiation formats to understand if they are supported and which is
```
Comment (akeranen) [inline draft-ietf-core-senml.md]: "negotiation formats"?

**Review by akeranen — CHANGES_REQUESTED** [review]

**Comment by cabo** [comment]
Assuming #120, we could add the appropriate references to {{sec-sensml}}.

**Comment by cabo** [comment]
Something is broken in the branch underlying this PR.  Abandoning; please use #121.

**Comment by akeranen** [comment]
Superseded by #121

**Pull request closed without merging**